app/server/src/core/check-env.ts

```diff
@@ -34,6 +34,10 @@ const isValidHost =
     // These values may change in the future
     (process.env.HOST === '0.0.0.0' || process.env.HOST === '127.0.0.1' || process.env.HOST === 'localhost');
 
+const isValidRedisHost = isString(process.env.REDIS_HOST);
+
+const isValidRedisPassword = isString(process.env.REDIS_PASSWORD);
```

Comment:
Should there be a different check for if it's production vs dev vs test?  Similar to the `HOST` variable.

Reply:
We don't use redis when running in development it's only really needed in production.